DSA-Campaign-Uplift-Estimation/src/main/webapp/About/about.html

```diff
@@ -0,0 +1,122 @@
+        <p>This project aims to address both of these issues by allowing advertisers to run test DSA campaigns with
+            easy-to-use UIs, so they can estimate the uplift DSA can bring to their keyword campaigns. Users can
+            experiment with different DSA campaign settings when creating their DSA campaigns, and the results of the
+            campaigns are displayed on the home and compare pages in clear, efficient charts and graphs. By allowing 
+            users to clearly see the value DSA can bring to their ad campaigns, this project hopes to convince more 
+            users to incorporate DSA into their ad campaigns.</p>
+        <p>This project is a model of what DSA could eventually implement. Many changes would have to be made for this
+            project to be used in production, such as a more accurate estimation feature, a cleaner, simpler UI, and 
+            auto-generated campaign settings using AI. This project would satisfy a very big need of advertisers today,
+            and if implemented, Google PMs predict it could potentially open a $750M market.</p>
+        <h3>User Experience</h3>
+        <p>When users access the site, they first have to login in order to view their created DSA campaigns. When they
+            access the home page, they will see information on all of their previously created DSA campaigns in the form 
+            of charts and graphs. Three test keyword campaigns have been created so that all users can interact with the 
+            UI and experiment with creating their own DSA campaigns. If implemented in production, the tool would pull keyword
+            campaigns from a user's existing Google Ads account.</p> 
+        <p>The create page is where users create and run DSA campaigns. When a DSA campaign is created, the campaign information
+            is listed as pending on the home page. After the estimation results are generated, the campaign status will change 
+            to complete, and the campaign results will be able to be viewed on the home and compare pages.</p>
+        <p>The compare page allows users to easily compare the results of their different DSA campaigns. Users can select
+            any number of their created DSA campaigns to compare, and the uplift statistics will be displayed along with 
+            the inputted campaign settings and generated search query reports.</p>
+        <h3>Settings Explanations</h3>
+        <ul>
+            <li>Start/End Dates (Optional) - used to calculate the campaign duration for uplift estimations. The default values
+                 for the start and end dates are the present date and one year in the future, respectively.</li>
+            <li>Manual CPC - how much to spend for every ad click.</li>
+            <li>Daily Budget - maximum amount to spend in a day.</li>
+            <li>Locations - locations to target. Currently only the US and US states are supported. Ex: display ads to users 
```

Comment:
maybe rephrase to: ads from the campaign will target users at the given locations. Currently only the US and US states are supported.

Reply:
done